### PR TITLE
v8

### PR DESCRIPTION
**Pull request by Keats**

### CODE REVIEW

**Comment by bergmark** [comment]
Hi, thanks for the library! Any plans to make a release out of this? I would very much appreciate it :-)

**Comment by Keats** [comment]
Just waiting for https://github.com/Keats/jsonwebtoken/issues/120 to be solved in a satisfying way. I won't have time for it in the short term so I will take PRs for it, as mentioned in the issue.

**Comment by rakshith-ravi** [comment]
@Keats - Just wanted to let you know: Thank you so much for maintaining this library. It makes it so much more easier for the ecosystem to build http servers that use a JWT authentication mechanism, thanks to this library. It's been working beautifully well so far and I just wanted to take a moment to appreciate the work you're doing :grimacing:

**Comment by Keats** [comment]
Thanks for the kind words @rakshith-ravi !

**Comment by benwis** [comment]
Hi @Keats! Thanks for maintaining this. I'd love to get v8 out so I can use the EdDSA algorithm without using the betas. Any chance you know when that is and is there anything I can help with?

**Comment by Keats** [comment]
Just need to handle https://github.com/Keats/jsonwebtoken/issues/190 and ideally https://github.com/Keats/jsonwebtoken/pull/119#issuecomment-867873222

**Comment by sebpuetz** [comment]
Hi @Keats, thanks for maintaining this great library! I was wondering if there were any blockers left for the v8 release and whether there's room for contributions to get rid of these blockers?

**Comment by Keats** [comment]
No blockers, beta 7 will likely be the v8 but it would be nice to have more feedback since there are a few breaking changes

**Comment by tiptenbrink** [comment]
@Keats any specific things that require extra feedback?

**Comment by Keats** [comment]
Just testing all the features and making sure everything works, I'd like to avoid having to release a major version three days after v8 release because I forgot one thing and it requires a breaking API change.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## 8.0.0 (unreleased)
+ 
+- Add EdDSA algorithm
+- `sign`/`verify` now takes a `&[u8]` instead of `&str` to be more flexible
+- `DecodingKey` now own its data
+- Remove deprecated `dangerous_unsafe_decode`
```
Comment (ionut-arm) [inline CHANGELOG.md:8]: Hello! I had a quick question: were the other, non-deprecated `dangerous_...` functions supposed to be removed from the API?

Comment (Keats) [reply CHANGELOG.md:8]: Yes, you can call https://github.com/Keats/jsonwebtoken/blob/next/src/validation.rs#L112-L114 to your Validation struct instead to achieve the same as the other `dangerous` fns

Comment (ionut-arm) [reply CHANGELOG.md:8]: Awesome, thanks!

**Pull request merged**